.travis.yml

```diff
@@ -0,0 +1,9 @@
+sudo: false
+
+install:
+  - sudo apt-get install jq
```

Comment:
Is `sudo` needed here? (I'm not sure)

Reply:
Yes, it's from the docs. I just tested without sudo and the build failed with Permissions denied ([#L440](https://travis-ci.org/openfaas/store/builds/422164035#L440))